### PR TITLE
Add module/components to existing resources CLI

### PR DESCRIPTION
**Pull request by DheerajGBhatt**
# Description
Add module/components to existing resources CLI.
## Issue ticket number and link
[Rov 59](https://linear.app/antstack/issue/ROV-59/add-modulecomponents-to-existing-resources-cli)

## Type of change

Please delete options that are not relevant.

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## New Packages

1.

### CODE REVIEW

**Review by alastairmoody-madeye — APPROVED** [review]
It is much more readable than before 🎉

File: src/.DS_Store
Comment (vishwasnavadak) [inline src/.DS_Store:1]: delete and add DS_Store to `.gitignore`?

File: src/configs/cliConfig.ts [hunk 1]
```diff
@@ -0,0 +1,55 @@
+import { Iroverdescription } from "../rover.types"
+import * as rover from "@rover-tools/engine/dist/bin/index"
+const rovercomponents = rover.components
```
Comment (vishwasnavadak) [inline src/configs/cliConfig.ts:3]: Camelcase?
```suggestion
const roverComponents = rover.components
```

File: src/roverCLI/roverDeployCLI.ts [hunk 2]
```diff
@@ -0,0 +1,83 @@
+import * as rover from "@rover-tools/engine/dist/bin/index"
+const rover_helpers = rover.helpers
```
Comment (vishwasnavadak) [inline src/roverCLI/roverDeployCLI.ts:2]: camelcase?

**Review by vishwasnavadak — CHANGES_REQUESTED** [review]

**Review by vishwasnavadak — APPROVED** [review]

**Pull request merged**